cylc/flow/workflow_db_mgr.py

```diff
@@ -405,9 +405,10 @@ def put_task_event_timers(self, task_events_mgr):
     def put_xtriggers(self, sat_xtrig):
         """Put statements to update external triggers table."""
         for sig, res in sat_xtrig.items():
-            self.db_inserts_map[self.TABLE_XTRIGGERS].append({
-                "signature": sig,
-                "results": json.dumps(res)})
+            if not sig.startswith('wall_clock(trigger_time='):
```

Comment:
Probably safer not to check the kwarg incase it's provided by position:

```suggestion
            if not sig.startswith('wall_clock('):
```